packages/query-core/src/queryObserver.ts

```diff
@@ -521,16 +521,17 @@ export class QueryObserver<
     }
 
     const isFetching = fetchStatus === 'fetching'
-    const isLoading = status === 'loading'
+    const isPending = status === 'pending'
     const isError = status === 'error'
 
     const result: QueryObserverBaseResult<TData, TError> = {
       status,
       fetchStatus,
-      isLoading,
+      isPending,
       isSuccess: status === 'success',
       isError,
-      isInitialLoading: isLoading && isFetching,
+      isInitialLoading: isPending && isFetching,
+      isLoading: isPending && isFetching,
```

Comment:
let's assign this to a `const isLoading = isPending && isFetching`, and then return `isInitialLoading: isLoading` as well as `isLoading` please